help.txt

```diff
@@ -12,13 +12,17 @@ For support issues not limited to the Android version of Anki, please visit the
 For questions and help concerning AnkiDroid, please visit the https://groups.google.com/forum/#!forum/anki-android[user forum], or send your questions to the forum's mailto:public-forum@ankidroid.org[public email address].
 
 == Bug Reports and Feature Requests
-For bug reports and feature requests, please search through the current list of open issues on the https://code.google.com/p/ankidroid/issues/list[AnkiDroid issue tracker], and if there are no existing issues, create a new issue including as much information as possible, including an error log whenever possible.
+For bug reports and feature requests, please search through the current list of open issues on the https://code.google.com/p/ankidroid/issues/list[AnkiDroid issue tracker], 
+and if there are no existing issues, create a new issue including as much information as possible. 
+Please also include the output of "debug info" from the AnkiDroid about dialog, which allows us to match your report with our internal crash report data.
```

Comment:
Does the description fit a case in which a user sets "Error reporting mode" to "Never report"?